Makefile

```diff
@@ -1,7 +1,7 @@
 PREFIX = /usr/local
 MANDIR = $(PREFIX)/share/man
 
-CDEBUGFLAGS = -Os -g -Wall
+CDEBUGFLAGS = -O0 -g -Wall
```

Comment:
If you're going to do this please make it non default somehow. -Os is very important for actual prod use as the routers have 16mb of storage total, a 400kb binary is a 1/32nd of all the available storage space.

Reply:
Fair enough, anyone needing this can just do `make CDEBUGFLAGS=-O0 -g -Wall`.

Reply:
But maybe it would be smart to move `-Os` and `-Wall` to regular CFLAGS and add a `debug` target with a separate flag set.

Reply:
make debug would be nicer for sure.